social-bot-manager/src/main/java/i5/las2peer/services/socialBotManagerService/SocialBotManagerService.java

```diff
@@ -614,6 +614,60 @@ public Response join(String body, @PathParam("botName") String botName) {
 			return Response.ok().entity(returnString).build();
 		}
 
+		/**
+		 * Endpoint that handles incoming webhook calls.
+		 * @param body JSONObject
+		 * @param botName Name of the bot.
+		 * @return HTTP response
+		 */
+		@POST
+		@Path("/{botName}/webhook")
```

Comment:
For me, webhook sounds very generic and more like you are setting up a webhook for a bot. Would it not be better to use a more intuitive name such as triggerChatEvent?

Reply:
Following the REST paradigm, I would tend to webhook. 
We should prevent function names in path definitions. 
Also chat_message would be one example to that is defined there, it would be possible to extend it and a bot could react on webhooks with service function calls or something else.